src/api.py

```diff
@@ -22,18 +32,30 @@ class PromptGenerationPlugin(Tagger):
     """
 
     class PromptGenerationPluginConfig(Config):
-        openai_api_key: str = Field("", description="An openAI API key to use. If left default, will use Steamship's API key.")
+        openai_api_key: str = Field("",
+                                    description="An openAI API key to use. If left default, will use Steamship's API key.")
         max_words: int = Field(description="The maximum number of words to generate per request")
```

Comment:
Do we want to limit this by default? e.g. only 400 tokens unless you increase it yourself?